src/components/AdvancedFeatures.js

```diff
@@ -9,8 +9,9 @@ export function AdvancedFeatures(props) {
   const isLoading = !(cards ?? []).every(
     (cardConfig) => cardConfig.isLoading == false
   );
-
-  if (isLoading)
+  let WCUnavialable = plugins?.status?.woocommerce !== "Active";
```

Comment:
Fix typo `WCUnavialable`. Should be `isWCUnavailable`. Check the same for other files as well